### PR TITLE
FixUI

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: navigation/index.tsx [hunk 1]
```diff
@@ -38,58 +46,117 @@ const MyTheme = {
...
+	
+			if(res.status === 200){
+				isLogin = true;
+				token = result.token;	
+				user = result.user;			
+			} else{
+				isLogin = false;
+			}
+		} catch (error) {
+			console.log(error.response);
```
Comment (teebarg) [inline navigation/index.tsx]: remove log

File: navigation/index.tsx [hunk 2]
```diff
@@ -38,58 +46,117 @@ const MyTheme = {
...
+		}
+		
+
+	} else {
+		alert('No values stored under that key.');
```
Comment (teebarg) [inline navigation/index.tsx]: Do we want to keep this?

File: screens/Otp.tsx [hunk 3]
```diff
@@ -43,26 +49,34 @@ export default function Otp({ navigation, route }) {
...
-					})
-					.catch((err) => {
-						let message = err?.response?.data?.message;
-						setErrorText(message);
-					})
-					.finally(() => {
-						setLoading(false);
-						return
-					});
+				// post(url, data)
```
Comment (teebarg) [inline screens/Otp.tsx]: remove comment if not needed

Comment (darmhoo) [reply screens/Otp.tsx]: done

File: navigation/index.tsx (hunk 1 above)
Comment (darmhoo) [reply navigation/index.tsx]: done

File: navigation/index.tsx (hunk 2 above)
Comment (darmhoo) [reply navigation/index.tsx]: no need to alert

**Review by teebarg — APPROVED** [review]

**Pull request merged**